setup.py

```diff
@@ -29,9 +29,11 @@
     keywords='workflow multithreaded rabbitmq',
     install_requires=[
         'frozendict',
-        'tornado>=4.1, <5.0',
         'pyyaml~=5.1.2',
         'pika>=1.0.0',
+        'nest_asyncio',
+        'aio-pika',
+        'aiocontextvars',
```

Comment:
Do we need any minimally required versions? If so would be good to use the `~=` compat operator

Reply:
No, I wouldn't expect you to do that. I just mentioned this in case you already happened to know for some packages that certain versions won't work. If not, fine to leave as is